source/blog/articles/2019-06-fast-and-furious.html.markdown

```diff
@@ -0,0 +1,94 @@
+
+[![Description](/assets/images/blog/2019/singing-and-chalk-art-700w.jpg)](/assets/images/blog/2019/singing-and-chalk-art-3000w.jpg)
+{: .article-image .article-image--has-caption}
+Open-air ministry in L'viv on Saturday afternoon.
+{: .caption-text .article-image__caption}
+
+Saturday in the morning we went touring! We went to the Armory Museum and then up to a lookout observatory on top of the highest point in L'viv. That was REALLY cool! In the afternoon we did street preaching, music, and chalk art. We also were able to hand out a lot of Good and Evil book tracts. Sunday we went and showed two film showings at the villages that we had handed out the invitations at on Thursday. Josh got sick Saturday night and was not able to go with us to show the films. We left at 10:30am Sunday morning to head to the villages. Since Josh always drives the van and he was sick, Ben drove. He only ran one red-light. 😆
+
+[![Description](/assets/images/blog/2019/nathan-preaching-700h.jpg)](/assets/images/blog/2019/nathan-preaching-3000h.jpg)
+{: .article-image .article-image--has-caption}
+Nathan preaches the Gospel at the conclusion of the film showing in Strilky.
+{: .caption-text .article-image__caption}
+
+Something cool that happened at the second village we showed the film at was that when we arrived at the place where we were to show the film at, it was still locked. We waited for a while, but no one came to unlock it we were trying to call our contacts, but no one was answering. Right about that time, two ladies and their kids came walking up to see the film. When they saw that we needed someone to come unlock the door, they started calling around to see if they could find somebody with a key. Sure enough they did, and within a half-hour the door was unlocked!  What was cool though is that every time we enter a village we are always the outsiders so to speak. But while they were helping us it was like they were on our side we were one of them! Also, I had so much fun playing with their little kids! Camping went well despite the fact that it rained most of the night. Thankfully, I don't think any of us got that wet. Today we went and handed film invitations for next Sunday at a village a little further down the road. We got that knocked out and then made it safely back to L'viv. We are all praying that God will heal Josh quickly so that he can be back with us. God is faithful. Onward we march to make Him known!
```

Comment:
In the second sentence, we might want to add a period to split the sentence after "no one came to unlock it". We might also want to change "today" towards the end of the paragraph to "Monday".

Reply:
Great! Done.